packages/journey-manager/src/index.ts

```diff
@@ -187,9 +187,10 @@ export interface RunProps {
    */
   ui?: UiConfig;
   /**
-   * The triggers dictionary, downloaded from the Dialog Studio desktop app
+   * The triggers dictionary, downloaded from the Dialog Studio desktop app.
+   * If triggers are not provided, they will be fetched from the CDN.
    */
-  triggers: Triggers;
+  triggers?: Triggers;
```

Comment:
Would it make more sense to pass in an explicit url rather than just null? That also gives them the opportunity to self-host if they so please?

Reply:
I think we should check with the broader team whether the custom hosting functionality should be an option (has implications on whether we want to support users with that).

The hosting we will do is automatically configured under those specified URL's at all times, what is optional here is the triggers themselves (mostly for backwards-compatibility), which they can get in a two-liner if they want.